pkg/morph/client/notary.go

```diff
@@ -751,19 +743,14 @@ func invocationParams(args ...interface{}) ([]sc.Parameter, error) {
 	return params, nil
 }
 
-// mn returns M and N multi signature numbers. For NeoFS N is a length of
-// inner ring list, and M is a 2/3+1 of it (like in dBFT). If committee is
-// true, returns M as N/2+1.
-func mn(ir []*keys.PublicKey, committee bool) (m int, n int) {
-	n = len(ir)
-
+// sigCount returns the number of required signature.
+// For NeoFS M is a 2/3+1 of it (like in dBFT).
```

Comment:
what does "For NeoFS" mean here? previously that was for `N` and described what the full multi-signature list is (the Alphabet)

Reply:
Fixed